Microsoft.HealthVault/Thing/ThingBase.cs

```diff
@@ -835,7 +835,9 @@ public string Serialize()
             XDocument newDocument = SDKHelper.SafeLoadXml(GetItemXml());
 
             string dataName = "thing/data-xml";
-            fetchedDocument.Element(dataName).ReplaceWith(newDocument.Element(dataName));
```

Comment:
>Element [](start = 28, length = 7)

what is cause for the change ? looks like if there is a problem then updates wont happen .. but that's not the case .. we are able to update ..